org-gcal.el

```diff
@@ -653,91 +489,48 @@ TO.  Instead an empty string is returned."
                           "please check/configure `org-gcal-file-alist'")
                   (buffer-name))))
 
+(defun org-gcal-start-date (time)
+    (if (< 11 (length time))
+      `("start" ("dateTime" .  ,time) ("date" .  nil))
+    `("start" ("date" .  ,time) ("dateTime" .  nil))))
+
+(defun org-gcal-end-date (time)
+  (if (< 11 (length time))
+      `("end" ("dateTime" .  nil) ("date" .  nil))
```

Comment:
This should have `("dateTime" . ,time)`, just like `org-gcal-start-date` above.

Reply:
ah, yes, nice catch. 
fixed